model/src/main/kotlin/Vulnerability.kt

```diff
@@ -51,58 +51,6 @@ data class Vulnerability(
             "A Vulnerability must have at least one reference."
         }
     }
-
```

Comment:
The commit message lacks a rationale. The rationale is:

As the severity is now stored in VulnerabilityReference, it also makes more sense to move the enums that map severity scores to ratings to that class.